samples/keys/identity_key_generation/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _identity_key_generate:
+
+Identity key generation
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to generate a random device-specific identity key and store in the Key
+Management Unit (KMU). The identity key is stored in the KMU in encrypted form using the Hardware
+Unique Key (HUK) Master Key Encryption Key (MKEK). The sample also demonstrates how to generate a
+random MKEK and store it in KMU.
+
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The identity key generation sample requires usage of :ref:`lib_hw_unique_key` library to generate
```

Comment:
```suggestion
This sample requires usage of :ref:`lib_hw_unique_key` library to generate
```

Reply:
Slightly modified version of this moved to "Requirements"-section